Editor/AGXUnityEditor/Windows/ConvertPhysXToAGXWindow.cs

```diff
@@ -0,0 +1,917 @@
+          agxSphere.Radius = physXSphere.radius * Mathf.Max( new float[]{
+                                                    Mathf.Abs(colliderScale.x),
+                                                    Mathf.Abs(colliderScale.y),
+                                                    Mathf.Abs(colliderScale.z)} );
+
+          agxSphere.IsSensor = physXSphere.isTrigger;
+
+          Undo.DestroyObjectImmediate( physXSphere );
+          break;
+
+        case "UnityEngine.BoxCollider":
+          var physXBox = asset.gameObject.GetComponent<BoxCollider>();
+          var agxBox = Undo.AddComponent<AGXUnity.Collide.Box>(newObject);
+
+          newObject.name = "AGXUnity.Collide.Box";
+          newObject.transform.localPosition = physXBox.center;
+
+          agxBox.HalfExtents = new Vector3( physXBox.size.x / 2f * Mathf.Abs( colliderScale.x ),
+                                           physXBox.size.y / 2f * Mathf.Abs( colliderScale.y ),
+                                           physXBox.size.z / 2f * Mathf.Abs( colliderScale.z ) );
+
+          agxBox.IsSensor = physXBox.isTrigger;
+
+          Undo.DestroyObjectImmediate( physXBox );
+          break;
+
+        case "UnityEngine.CapsuleCollider":
+          var physXCapsule = asset.gameObject.GetComponent<CapsuleCollider>();
+          var agxCapsule = Undo.AddComponent<AGXUnity.Collide.Capsule>(newObject);
+
+          newObject.name = "AGXUnity.Collide.Capsule";
+          newObject.transform.localPosition = physXCapsule.center;
+
+          // Capsule radius is old radius times abs largest xz-localScale axis. PhysX capsule height is including caps, agx is without
+          newObject.transform.localScale = Vector3.one;
+          var radius = physXCapsule.radius * Mathf.Max(new float[]{
+                                                      Mathf.Abs(colliderScale.x),
+                                                      Mathf.Abs(colliderScale.z)});
+          agxCapsule.Radius = radius;
+          agxCapsule.Height = ( physXCapsule.height - radius * 2 ) * Mathf.Abs( colliderScale.y );
+
+          agxCapsule.IsSensor = physXCapsule.isTrigger;
+
+          Undo.DestroyObjectImmediate( physXCapsule );
+          break;
+
+        case "UnityEngine.MeshCollider":
+          var physXMesh = asset.gameObject.GetComponent<MeshCollider>();
+          var agxMesh = Undo.AddComponent<AGXUnity.Collide.Mesh>(newObject);//
+
+          newObject.name = "AGXUnity.Collide.Mesh";
+          newObject.transform.localPosition = Vector3.zero;
+
+          if (physXMesh.sharedMesh.isReadable)
+          {
+            agxMesh.AddSourceObject( physXMesh.sharedMesh );
+          }
+          else
+          {
+            Mesh readableMesh = new Mesh();
+            readableMesh.vertices = physXMesh.sharedMesh.vertices;
+            readableMesh.triangles = physXMesh.sharedMesh.triangles;
+            agxMesh.AddSourceObject( readableMesh );
```

Comment:
Does this case work across reloads and such? I vaguely remember trying something similar and it not yielding the desired results